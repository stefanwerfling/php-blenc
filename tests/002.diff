001+ Fatal error: Validation of script 'ext/blenc/tests/output.phpe' failed, cannot execute. in /home/john/working/php-src/ext/blenc/tests/002.php on line 6
001- Hello, World!
002+ /home/john/working/php-src/main/streams/streams.c(374) : Stream of type 'STDIO' 0x401f9dc4 (path:ext/blenc/tests/output.phpe) was not closed
003+ /home/john/working/php-src/Zend/zend_hash.c(1276) : ht=0x824d2fc is already destroyed