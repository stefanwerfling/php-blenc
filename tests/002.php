<?php 

    ini_set("blenc.key_file", "ext/blenc/tests/keys.txt");
    $key = blenc_encrypt(file_get_contents("input.php"), "ext/blenc/tests/output.phpe");
    file_put_contents("ext/blenc/tests/keys.txt", $key);
    include_once("ext/blenc/tests/output.phpe");

    unlink("ext/blenc/tests/keys.txt");
    unlink("ext/blenc/tests/output.phpe");

?>
