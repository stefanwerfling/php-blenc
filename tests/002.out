Fatal error: Validation of script 'ext/blenc/tests/output.phpe' failed, cannot execute. in /home/john/working/php-src/ext/blenc/tests/002.php on line 6
/home/john/working/php-src/main/streams/streams.c(374) : Stream of type 'STDIO' 0x401f9dc4 (path:ext/blenc/tests/output.phpe) was not closed
/home/john/working/php-src/Zend/zend_hash.c(1276) : ht=0x824d2fc is already destroyed